core/src/test/java/com/linecorp/armeria/client/HttpClientSniTest.java

```diff
@@ -97,53 +95,55 @@ public static void destroy() throws Exception {
     }
 
     @Test
-    public void testMatch() throws Exception {
+    void testMatch() throws Exception {
         testMatch("a.com");
         testMatch("b.com");
     }
 
     private static void testMatch(String fqdn) throws Exception {
-        assertEquals(fqdn + ": CN=" + fqdn, get(fqdn));
+        Assertions.assertEquals(fqdn + ": CN=" + fqdn, get(fqdn));
     }
 
     @Test
-    public void testMismatch() throws Exception {
+    void testMismatch() throws Exception {
         testMismatch("127.0.0.1");
         testMismatch("mismatch.com");
     }
 
     private static void testMismatch(String fqdn) throws Exception {
-        assertEquals("b.com: CN=b.com", get(fqdn));
+        Assertions.assertEquals("b.com: CN=b.com", get(fqdn));
     }
 
     private static String get(String fqdn) throws Exception {
         final HttpClient client = HttpClient.of(clientFactory, "https://" + fqdn + ':' + httpsPort);
 
         final AggregatedHttpResponse response = client.get("/").aggregate().get();
 
-        assertEquals(HttpStatus.OK, response.status());
+        Assertions.assertEquals(HttpStatus.OK, response.status());
         return response.contentUtf8();
     }
 
     @Test
-    public void testCustomAuthority() throws Exception {
-        final HttpClient client = HttpClient.of(clientFactory, "https://127.0.0.1:" + httpsPort);
-        final AggregatedHttpResponse response =
-                client.execute(RequestHeaders.of(HttpMethod.GET, "/",
-                                                 HttpHeaderNames.AUTHORITY, "a.com:" + httpsPort))
```

Comment:
`:authority` header will always be automatically set based on the specified or selected `Endpoint`, so you cannot set an `:authority` header manually anymore, although you can still send the `:authority` header you desire, by specifying an `Endpoint` explicitly.